docs/nodes/mocha-testnet.mdx

```diff
@@ -55,6 +55,7 @@ Below is a list of RPC endpoints you can use to connect to Mocha testnet:
 
 * `rpc-mocha.pops.one`
 * `rpc.mocha.celestia.counterpoint.software`
+* [https://celestia-testnet-rpc.polkachu.com](https://celestia-testnet-rpc.polkachu.com)
```

Comment:
is this expected?

https://celestia-testnet-rpc.polkachu.com/block?height=_

```json
{
"jsonrpc": "2.0",
"id": -1,
"error": {
"code": -32602,
"message": "Invalid params",
"data": "error converting http params to arguments: invalid 64-bit integer encoding \"_\", expected string"
}
}
```

Reply:
please format this the same as others, as well, thanks!

Reply:
Just want to reply to two questions.
1. You need to specify a block number if you want to query a block. For example, https://celestia-testnet-rpc.polkachu.com/block?height=65420 It is expected to receive an error when you do not provide a block number. 
2. A new PR is submitted to fix the format issue. Thanks.

Reply:
:smile: thank you for the context!

Reply:
🤦